css/AlookA.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Open+Sans:wght@300;400&family=Poppins:wght@400;600;900&family=Roboto:wght@400;700&display=swap');
```

Comment:
We are using your code for a better purpose together. Thank you for your contribution :-)